.freeipa-pr-ci.yaml

```diff
@@ -58,11 +58,23 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-28/build_url}'
-        test_suite: test_integration/test_external_ca.py::TestExternalCA test_integration/test_external_ca.py::TestSelfExternalSelf test_integration/test_external_ca.py::TestExternalCAInstall
+        test_suite: test_integration/test_external_ca.py::TestExternalCA
```

Comment:
Does it make sense to add another test from the external CA test suite here?

Reply:
That test takes 17 to 20 minutes by itself. The split distributes the load evenly across two jobs.

Reply:
Fair enough